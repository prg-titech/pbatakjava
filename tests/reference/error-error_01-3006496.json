{
    "basename": "error-error_01-3006496",
    "cmd": "{infile}",
    "infile": "tests/error_01/build.sh",
    "infile_hash": "e27a56233474522720c6bbda440b67720f411838a5deb23f992278d2",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "error-error_01-3006496.stderr",
    "stderr_hash": "0d1dd2123b90ad56ceafef9208e174ac0cd79a1aa3aea9cbbff59867",
    "returncode": 4
}