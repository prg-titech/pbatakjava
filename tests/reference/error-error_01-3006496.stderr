tests/error_01/Test.java:1: error: no solution found in this component
